File: Day1/217-Contains-Duplicate/Method1.cpp
class Solution
{
public:
    bool containsDuplicate(vector<int> &nums)
    {
        unordered_set<int> st;

        for (const auto &n : nums)
        {
            if (st.count(n))
                return true;
            st.insert(n);
        }
        return st.size() != nums.size();
    }
};

File: Day1/26-Remove-Duplicates-from-Sorted-Array/Method1.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int j = 0, count = 1;

        for (int i = 1; i < nums.length; i++) {
            if (nums[i] == nums[j])
                continue;
            else {
                j++;
                count++;
                nums[j] = nums[i];
            }
        }
        return count;
    }
}